backend/EkoFunkcje/EkoFunkcje/Features/Attachments/AddAttachmentFunction.cs

```diff
@@ -0,0 +1,36 @@
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace EkoFunkcje.Features.Attachments
+{
+    public static class AddAttachmentFunction
+    {
+        [FunctionName("AddAttachment")]
+        public static async Task<HttpResponseMessage> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "AddAttachment/{interventionId}/{fileName}")] HttpRequestMessage req,
+            [Blob("http://127.0.0.1:10000/devstoreaccount1/attachments/12345/av-2102622222.jpg", FileAccess.Write , Connection = Config.StorageConnectionName)] CloudBlockBlob newBlob,
```

Comment:
Co to za sztywny adres?

Reply:
ta nazwa nie powinna byc jakos autogenerowana na podstawie numeru zgloszenia i jakiegos hasha?

Reply:
okej rozumiem ze one wszystkie beda trzymane w tym katalogu wobec tego tylko pobierz jakos nazwe pliku z wuadomosci i te ustaw albo ja wygeneruj

Reply:
Jakiś syf został, poprawione